config/src/config/network_config.rs

```diff
@@ -194,12 +182,11 @@ impl NetworkConfig {
         } else {
             PeerId::try_from(network_keypairs.identity_keys.public().to_bytes()).unwrap()
         };
-        self.network_keypairs = network_keypairs;
+        self.network_keypairs = Some(network_keypairs);
     }
 
     pub fn set_default_peer_id(&mut self) {
-        self.peer_id =
-            PeerId::try_from(self.network_keypairs.identity_keys.public().to_bytes()).unwrap();
+        self.peer_id = PeerId::default();
     }
```

Comment:
Probably should just delete this function now that it doesn't have complex logic within it.

Reply:
I was initially reluctant because it reveals NetworkConfig internals, but the only place where it's used is in testing, so that should be acceptable (there's already a lot of internal manipulation happening anyway..) -- done.